bin/amphtml-update.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+# Go to the right location.
+if [[ '.' != $(dirname "$0") ]]; then
+	cd bin
+fi
+
+BIN_PATH="$(pwd)"
+PROJECT_PATH=$(dirname $PWD)
+VENDOR_PATH=$PROJECT_PATH/vendor
+
+if ! apt-get > /dev/null; then
+	echo -e "The AMP HTML uses the apt-get, make sure to run this script in a Linux environment"
+	exit
+fi
+
+# Install dependencies.
+sudo apt-get install git
+sudo apt-get install python
+sudo apt-get install protobuf-compiler
+sudo apt-get install python-protobuf
+
+# Create and go to vendor.
+if [[ ! -e $VENDOR_PATH ]]; then
+	mkdir $VENDOR_PATH
+fi
+cd $VENDOR_PATH
+
+# Clone amphtml repo.
+if [[ ! -e $VENDOR_PATH/amphtml ]]; then
+	git clone https://github.com/ampproject/amphtml amphtml
+fi
```

Comment:
Maybe add an `else` here which goes into `$VENDOR_PATH/amphtml` and does `git pull`?